### ISSUE TITLE
Sample SIRE data doesn't process properly

### ISSUE THREAD

**Issue by davidlmobley** [MobleyLab/alchemical-analysis#54]
I ran the latest master alchemical_analysis on the sire data under samples as per the relevant README, and ran into the below error. @halx , can you have a look?

```
Loading in data from data/actual_grad_0000.dat  (state 0) ...
Traceback (most recent call last):
  File "../../alchemical_analysis/alchemical_analysis.py", line 1224, in <module>
    main()
  File "../../alchemical_analysis/alchemical_analysis.py", line 1169, in main
    nsnapshots, lv, dhdlt, u_klt = parser_sire.readDataSire(P)
  File "/Users/dmobley/GoogleDrive/research/github/alchemical-analysis/alchemical_analysis/parser_sire.py", line 90, in readDataSire
    f.loadtxtSire(nf)
  File "/Users/dmobley/GoogleDrive/research/github/alchemical-analysis/alchemical_analysis/parser_sire.py", line 63, in loadtxtSire
    dhdlt[state, :, :nsnapshots[state]] = numpy.genfromtxt(self.filename, dtype=float, skiprows=self.skip_lines, usecols=1)
TypeError: genfromtxt() got an unexpected keyword argument 'skiprows'
```

Actually, a quick Google suggests that this is a problem with an updated numpy. I may be able to quickly fix. Specifically:
```
skiprows : int, optional
skiprows was removed in numpy 1.10. Please use skip_header instead.
```

**Comment by davidlmobley** [comment]
Fixed by #55

**Comment by halx** [comment]
I'm confused.  The error message suggest that this is happening in the Sire
parser, not the AMBER parser.  Yes, it looks like a numpy problem.

**Comment by davidlmobley** [comment]
Yes, my first post incorrectly said AMBER. It was SIRE. HOwever, I fixed it in #55.
